### DISCUSSION TITLE
get the return code of getaddrinfo?

### DISCUSSION THREAD

**Discussion by yushicheng7788** [curl/curl#13865] [Q&A] [answered]
Hi, when CURLcode is **CURLE_COULDNT_RESOLVE_HOST**, Can I have some methods to get the return code of getaddrinfo system call? I noticed that this value was stored at data->state.async.status, but I have no ways to fetch it.

**Comment by bagder** [comment, marked as answer]
No, there is no way to get that value. In fact, libcurl might not even use getaddrinfo as it depends on how it was built and where it runs.

**Reply by yushicheng7788** [reply]
Thanks for your response.
